File: benches/baselines/micro/loop_sum.js
// 10M-iteration arithmetic loop
const t0 = Date.now();
let acc = 0;
let i = 0;
while (i < 10000000) {
    acc = acc + i % 7;
    i = i + 1;
}
const t1 = Date.now();
print_line(`RESULT ${acc} MS ${t1 - t0}`);

File: benches/baselines/micro/loop_sum.lua
-- 10M-iteration arithmetic loop
local t0 = os.clock()
local acc = 0
local i = 0
while i < 10000000 do
  acc = acc + i % 7
  i = i + 1
end
local t1 = os.clock()
print(string.format("RESULT %d MS %d", acc, math.floor((t1 - t0) * 1000)))
